hexa/datasets/models.py

```diff
@@ -357,6 +358,8 @@ def encode(self, obj):
         def replace_nan(item):
             if isinstance(item, float) and math.isnan(item):
                 return None
+            elif isinstance(item, bytes):
+                return base64.b64encode(item).decode("utf-8")
             elif isinstance(item, dict):
```

Comment:
[Non-blocking] It's a bit confusing that there is actually no nan replacement for bytes as the name of the method suggests, I would maybe expect something like :

```suggestion
            elif isinstance(item, str) and item.lower() == "nan":
                return None
            elif isinstance(item, bytes):
                return replace_nan(base64.b64encode(item).decode("utf-8"))
```

Maybe I am just overcomplicating here :)